README.md

```diff
@@ -41,38 +44,46 @@
 - **[Users can create a category.]**
 - **[Limit access to web app resources based on authorization rules.]**
 
-- Demo Link [Live Demo Link]()
-- Walkthrough[Walkthrough video]()
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-app-nkic.onrender.com)
```

Comment:
- [optional] Great job successfully deploying, however app encounters issue when image with PNG format is added instead of JPEG. Kindly look into that.

Reply:
@tufoinnkuo10 I will fix that. Thank you